### DISCUSSION TITLE
Navigation menu broken with Firefox Android since today

### DISCUSSION THREAD

**Discussion by lazka** [community/community#75847] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The menu opens outside of the viewport and many of the entries can't be clicked.
Reproducible with Firefox stable and nightly.

![IMG_20231117_084011](https://github.com/community/community/assets/991986/a691595d-c2c2-4e0c-a82d-32a3e4b74620)

**Comment by codezen879** [comment]
fine1

**Comment by codezen879** [comment]
fine2

**Comment by sigprof** [comment]
Can also be reproduced on desktop Firefox 119.0.1 in the Adaptive Design mode (Ctrl+Shift+M):
![image](https://github.com/community/community/assets/616082/a699d3e0-6ac9-43fd-adb9-fc451638111f)

**Comment by lazka** [comment]
Looks like it got fixed just now

**Reply by indrastorms** [reply]
Nope

**Reply by lazka** [reply]
right, it's only fixed on Android, still broken on the desktop

**Reply by indrastorms** [reply]
Using firefox beta android and the issue is still there.

**Reply by lazka** [reply]
Weird. I'm using firefox nightly on Android.

**Reply by yan12125** [reply]
Thanks! I can confirm this is fixed on nightly. Still an issue on beta, though.
